src/index.js

```diff
@@ -10,7 +10,7 @@ const context = (state) => ({ tests:[], before:[], after:[], bEach:[], aEach:[],
 const milli = arr => (arr[0]*1e3 + arr[1]/1e6).toFixed(2) + 'ms';
 const hook = (ctx, key) => handler => ctx[key].push(handler);
 
-if (isNode = typeof process !== 'undefined') {
+if (isNode = typeof process !== 'undefined' && !process.browser) {
```

Comment:
I don't think `browser` is a valid property. That is something that NextJs added afaik. The mere presence of `process` should be enough to detect Node vs. Browser.

Reply:
Agreed – this a bundler and/or build-tool replacement/assignment, and doesn't have anything to do with Node.js itself.

I included a stricter check as part of #63 that should work with Snowpack too.